server/tserver/src/main/java/org/apache/accumulo/tserver/log/TabletServerLogger.java

```diff
@@ -458,6 +458,26 @@ private void write(final Collection<CommitSession> sessions, boolean mincFinish,
       // the logs haven't changed.
       final int finalCurrent = currentLogId;
       if (!success) {
+
+        if (tserver.getLock() == null || !tserver.getLock().verifyLockAtSource()) {
```

Comment:
The same thing applies here if my other [comment](https://github.com/apache/accumulo/pull/5169#discussion_r1884214223) is a valid concern

Reply:
applied in 77492bf